aws_lambda_powertools/utilities/batch/sqs_fifo_partial_processor.py

```diff
@@ -12,6 +15,14 @@ class SQSFifoCircuitBreakerError(Exception):
     pass
 
 
+class SQSFifoMessageGroupCircuitBreakerError(Exception):
+    """
+    Signals a record not processed due to the SQS FIFO message group processing being interrupted
+    """
+
+    pass
```

Comment:
Do we need `pass` here?

Reply:
If you look a little bit further up, this is just a copy of the `SQSFifoCircuitBreakerError` exception

Reply:
Aa ok, good! Thanks.